makemessages.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+cmd="python manage.py makemessages --all --no-wrap --no-location --no-obsolete"
```

Comment:
```suggestion
cmd="python3 manage.py makemessages --all --no-wrap --no-location --no-obsolete"
```

Not all environments might contain "python" in path, but "python3" was present all the time.